editoast/justfile

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env just --justfile
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Run the application; does not start other services
+run:
+    cargo run
+
+# Install required dependencies
+install:
+    cargo build
+
+# Run the tests
+test:
+    cargo test -- --test-threads=4
```

nitpick: Why the need for `--test-threads`? I would just remove it. If you really need it, then what about this instead? (or through an environment variable)
```suggestion
test args:
    cargo test -- {{args}}
```

---

that’s what the readme says, should we change the readme aswell? Is it a decision that should be made in this pull request?

---

The number of threads is now a by default 4, but can be overriden (and just allows to pass any arbitatry arguments after)